src/Raven.Server/Documents/Operations/Operations.cs

```diff
@@ -95,6 +95,7 @@ public Operation GetOperation(long id)
         }
 
         public Task<IOperationResult> AddOperation(
+            string databaseName,
```

Comment:
Let's add this as an optional parameter

Reply:
By default it will be `null` and we'll set it accordingly for the operation that we need.

Reply:
Done